http/1.5/modules/ROOT/pages/http-troubleshooting.adoc

```diff
@@ -92,6 +97,86 @@ In this example, an HTTP request operation is configured `requestStreamingMode`
 </flow>
 ----
 
+== Understand Common Throws
+
+Here is a list of common throws messages and how to interpret them.
+
+* HTTP:BAD_REQUEST
+
+ The server can not understand the request due to invalid syntax. For example, when some header size exceeds the maximum.
+
+* HTTP:CLIENT_SECURITY
+
+ Indicates a security type problem enforced by an external entity.
```

Comment:
Reword to make this description parallel with the others (noun-based rather than verb-based):
```suggestion
 A security problem enforced by an external entity occurred.
```